documentation/docs/01-routing.md

```diff
@@ -277,17 +277,61 @@ A route can have multiple dynamic parameters, for example `src/routes/[category]
 
 > `src/routes/a/[...rest]/z.svelte` will match `/a/z` as well as `/a/b/z` and `/a/b/c/z` and so on. Make sure you check that the value of the rest parameter is valid.
 
-#### Fallthrough routes
+#### Sorting
+
+It's possible for multiple routes to match a given path. For example each of these routes would match `/foo-abc`:
+
+```bash
+src/routes/[a].js
+src/routes/[b].svelte
+src/routes/[c].svelte
+src/routes/[...catchall].svelte
+src/routes/foo-[bar].svelte
+```
 
-Finally, if you have multiple routes that match a given path, SvelteKit will try each of them until one responds. For example if you have these routes...
+SvelteKit needs to know which route is being requested. To do so, it sorts them according to the following rules...
+
+- More specific routes are higher priority
```

Comment:
it's a bit unclear here whether a more specific page would beat a less specific standalone endpoint

Reply:
it's immediately followed by an example that should eliminate any ambiguity though, no?

Reply:
It's a lot harder to reverse engineer the rules from an example. Why not just state it up front?